metron-analytics/metron-profiler-storm/src/test/java/org/apache/metron/hbase/bolt/HBaseBoltTest.java

```diff
@@ -67,18 +65,15 @@ public void setupTuples() throws Exception {
     when(tuple2.getValueByField(eq("widget"))).thenReturn(widget2);
   }
 
-  @Before
-  public void setup() throws Exception {
-    tuple1 = mock(Tuple.class);
-    tuple2 = mock(Tuple.class);
-    client = mock(HBaseClient.class);
-    provider = mock(TableProvider.class);
+  @BeforeEach
+  public void setup() {
```

Comment:
Looks like the redundant setup method should be removed entirely?

Reply:
Dropped it.